income.json

```
@@ -17,13 +17,15 @@
       "cooldown": 9000000,
       "payout": {
         "min": 700,
-        "max": 1500
+        "max": 2050
       },
       "success": [
         "Вы так хорошо работали, что заработали %amount%",
         "Хорошая работа, Олег! На этот раз вы смогли заработать %amount%. Не так уж и много, правда?",
         "Грузчик - не такая уж и плохая профессия, правда платят меньше чем хотелось бы. Всего лишь %amount%",
         "Да вы счастливчик. Работодатель не видел, как вы бездельничали и заплатил вам %amount%"
```

После каждого ответа должна быть запятая
```suggestion
        "Да вы счастливчик. Работодатель не видел, как вы бездельничали и заплатил вам %amount%",
```